src/frontend/apps/lti_site/components/VideoWizard/CreateVOD/index.spec.tsx

```diff
@@ -140,6 +140,68 @@ describe('<CreateVOD />', () => {
     expect(createVideoButton).toBeDisabled();
   });
 
+  it('renders CreateVOD with video title already set', async () => {
+    const mockedVideo = videoMockFactory({
+      id: 'videoId',
+      title: 'Video title already set',
+      description: null,
+      upload_state: uploadState.PENDING,
+      is_ready_to_show: false,
+    });
+
+    mockedUseVideo.mockReturnValue(mockedVideo);
+
+    mockedUseAppConfig.mockReturnValue({
+      static: {
+        img: {
+          videoWizardBackground: 'img/path/videoWizardBackground',
+        },
+      },
+    } as any);
+
+    fetchMock.mock(
+      `/api/videos/`,
+      {
+        actions: { POST: { license: { choices: licenseChoices } } },
+      },
+      { method: 'OPTIONS' },
+    );
+
+    mockUseUploadManager.mockReturnValue({
+      addUpload: jest.fn(),
+      resetUpload: jest.fn(),
+      uploadManagerState: {},
+    });
+
+    render(<CreateVOD video={mockedVideo} />);
+
+    screen.getByText('Video creation');
+    screen.getByText(
+      'Use this wizard to create a new video, that you will be able to share with your students.',
+    );
+
+    const titleInput: HTMLInputElement = screen.getByRole('textbox', {
+      name: 'Enter title of your video here',
+    });
+    expect(titleInput.value).toEqual(mockedVideo.title);
+    screen.getByText('Add a video or drag & drop it');
```

Comment:
I am not sure what should be the standard.

```suggestion
   expect(
     screen.getByText('Add a video or drag & drop it'),
    ).toBeInTheDocument();
```

Reply:
We can progressively migrate to this standard in our new tests or when we update existing.